src/Config/Schema.php

```diff
@@ -79,7 +79,7 @@ public function getSchema()
                 self::SCHEMA_VALUE_VALIDATION => function (string $key, $value) {
                     if (!in_array($value, range(0, 9))) {
                         return sprintf(
-                            'The SCD_COMPRESSION_LEVEL variable contains an invalid value of type string. ' .
+                            'The %s variable contains an invalid value #%d. ' .
```

Comment:
The `#` is really unnecessary here, I would leave it off

Reply:
agree, removed